engine/runtime/src/main/java/org/enso/interpreter/node/callable/ExecuteCallNode.java

```diff
@@ -60,7 +60,8 @@ protected Object callInlineable(
       Object[] arguments,
       @Cached("function.getCallTarget()") RootCallTarget cachedTarget,
       @Cached("createInlineableNode(cachedTarget)") InlineableNode callNode) {
-    var args = Function.ArgumentsHelper.buildArguments(function, callerInfo, state, arguments);
+    var args =
+        Function.ArgumentsHelper.buildArguments(function, callerInfo, arguments); // XXX no state
```

Comment:
What does this comment mean?

Reply:
Removed in 13789d323d55603641b99e3757950ad519bc9fd8 - there should be no more XXX comments left now.